docs/solutions/rhel74_image/datasources.tf

```diff
@@ -0,0 +1,59 @@
+# DO NOT ALTER THIS FILE
+
+data "oci_identity_compartments" "compartment" {
+	compartment_id = "${var.tenancy_ocid}"
+	filter {
+		name = "name"
+		values = [ "${var.build_env["compartment"]}" ]
+	}
+}
+
+data "oci_identity_availability_domains" "ad" {
+	compartment_id = "${var.tenancy_ocid}"
+	filter {
+		name = "name"
+		values = [ "\\w*-${upper(var.build_env["ad"])}" ]
+		regex = true
+	}
+}
+
+data "oci_core_virtual_networks" "vcn" {
+	compartment_id = "${data.oci_identity_compartments.compartment.compartments.0.id}"
+	filter {
+		name = "display_name"
+		values = [ "${var.build_env["vcn"]}" ]
+	}
+}
+
+data "oci_core_subnets" "subnet" {
+	compartment_id = "${data.oci_identity_compartments.compartment.compartments.0.id}"
+	vcn_id = "${data.oci_core_virtual_networks.vcn.virtual_networks.0.id}"
+	filter {
+		name = "display_name"
+		values = [ "${var.build_env["subnet"]}" ]
+	}
+
+}
+
+data "oci_core_images" "image" {
```

Comment:
This method of getting the image OCID is used in most of our examples, but it will be changing in the near future. For now, we have the following comment on each usage - please add this comment here as well:

# Gets the OCID of the image. This technique is for example purposes only. The results of oci_core_images may
# change over time for Oracle-provided images, so the only sure way to get the correct OCID is to supply it directly.